.github/workflows/check-md-links.yml

```diff
@@ -0,0 +1,22 @@
+name: Check Markdown links
+# https://github.com/gaurav-nelson/github-action-markdown-link-check
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+    types: [opened, reopened, ready_for_review, synchronize]
+    paths:
+      - ".github/workflows/md-check-links.yml"
+      - "**/*.md"
```

Comment:
Mind using a `workflow_call` trigger just like this?
https://github.com/Lightning-AI/utilities/blob/face8f06c7dd81db2991e2f8e815e0b300340081/.github/workflows/check-schema.yml#L4

(By having `workflow_call` trigger type, this `check-md-links.yml` workflow can be called from other workflows!)

Also, could you reuse this workflow in this calling workflow?
https://github.com/Lightning-AI/utilities/blob/face8f06c7dd81db2991e2f8e815e0b300340081/.github/workflows/ci-use-checks.yml

Reply:
Can you check now?